Assets/Scripts/WinningStack.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Assets.Scripts;
+
+public class WinningStack : CardStack
+{
+    private Card.Value lastCardValue;
+    private Card.Suit stackSuit;
+
+    public bool IsCompleted()
+    {
+        return lastCardValue == Card.Value.King;
+    }
+
+    public bool CanDrop(Card cardToDrop)
```

Comment:
This looks good, and I think it will get the job done. We could wrap this in some unit tests and then refactor it down to some fewer lines if we wanted to.